Update README.md (#25)
pinojs/pino-elasticsearch: README.md

You can then use [Kibana](https://www.elastic.co/products/kibana) to
browse and visualize your logs.

> ## Used as a module

Reviewer: ```suggestion
## Use as a module
```

> use pino-elasticsearch as a module is sample, use pino-multi-stream to send log to multi transport, for example:

Reviewer: ```suggestion
use pino-elasticsearch as a module is simple, use [pino-multi-stream](https://www.npmjs.com/package/pino-multi-stream) to send log to multi transport, for example:
```

> ```

Reviewer: ```suggestion
```js
```

const pinoms = require('pino-multi-stream')
const pinoEs = require('pino-elasticsearch')({
    host: '192.168.1.220',
    index: 'zb',
    port: '9200'
})

const logger = pinoms({
    streams: [
      {level: 'error', stream: process.stderr}, // an "error" level destination stream
      {level: 'info', stream: process.stdout}, // an "info" level destination stream
      {stream: pinoEs}
    ]
  })

logger.info({'msg': {'info': 'info'}})
logger.debug('debug')
logger.warn('warn')
logger.error('error')

```

> *** Notice, create the pino-elasticsearch must set host & port ***

Reviewer: ```suggestion

*** Notice, the `host` and `port` parameters of `pino-elasticsearch` are required ***
```
